lib/modelHelpers.py

```diff
@@ -53,22 +53,33 @@ def generate_payload(
         "Authorization": f"Bearer {TOKEN}",
     }
 
+    message = {"type": "text", "text": content}
+    if content == "clip":
+        clipped_image = clip.image()
+        if clipped_image:
+            data = clipped_image.encode().data()
+            base64_image = base64.b64encode(data).decode("utf-8")
+            message = {
+                "type": "image_url",
+                "image_url": {"url": f"data:image/jpeg;base64,{base64_image}"},
+            }
+        else:
+            message = {"type": "text", "text": clip.text()}
+
     data = {
         "messages": [
             {
                 "role": "system",
                 "content": settings.get("user.model_system_prompt")
                 + additional_context,
             },
-            {"role": "user", "content": f"{prompt}:\n{content}"},
+            {"role": "user", "content": [{"type": "text", "text": prompt}, message]},
         ],
         "max_tokens": 2024,
         "temperature": settings.get("user.model_temperature"),
         "n": 1,
-        "stop": None,
```

Comment:
This was causing an exception when running with an image prompt but not with text. I didn't notice any behavioral difference however so I removed it.

Reply:
Good w me if it works with you